docs/samples/contact-center/app.js

```
@@ -557,59 +549,136 @@ function registerTaskListeners(task) {
+    disableAllCallControls();
+  } else if (task.data.interaction.mediaType === 'chat' || task.data.interaction.mediaType === 'email') {
+    holdResumeElm.disabled = true;
+    muteElm.disabled = true;
+    pauseResumeRecordingElm.disabled = true;
+    consultTabBtn.disabled = true;
+    declineElm.disabled = true;
+    transferElm.disabled = false;
+    endElm.disabled = !hasParticipants;
+  } else if (task?.data?.interaction?.mediaType === 'telephony') {
+    // hold/resume call
+    const isHold = media && media[mediaResourceId] && media[mediaResourceId].isHold;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)